carHandler.py

```diff
@@ -0,0 +1,61 @@
+import csv
+import json
+import os
+from uuid import uuid4
+
+with open('input.csv', 'r') as csv_file:
+    rows = csv.reader(csv_file, delimiter=';')
+    cars_data = list(rows)
+    keys = list(cars_data[0])
+    keys.insert(0, 'UUID')
+    keys = tuple(keys)
+    values = cars_data[1:]
+    for value in values:
+        uuid = str(uuid4())
+        value.insert(0, uuid)
+    cars = []
+    for item in cars_data:
```

Comment:
Do you really have to parse the data twice? Didn't you already parsed those before?